packages/react-vapor/src/components/radio/examples/RadioExamples.1.use.md

```diff
@@ -4,29 +4,25 @@
 
 ### General
 
--   A set should always have a title.
--   Use a wording that indicates that the user can select **only one option**. Some user may not know the difference between radio buttons and checkboxes.
--   Try to put the most common options at the top of the list, unless your options have a strong logical sorting order (such as prices or sizes).
--   Keep the set under 7 options to choose from.
+-   An option set should always have a title.
+-   Use wording that indicates that the user can **only select one option**. Some users may not know the difference between radio buttons and checkboxes.
+-   Put the most common options at the top of the list, unless your options have a strong logical ordering (such as prices or sizes).
+-   Keep the number of options per set under 7.
 
 ### Label
 
--   Try to limit the size of the label to one short line.
--   Add a description or help hint only if necessary to explain the option.
+-   Try to limit label text to one short line.
+-   Add a description or help hint only if necessary.
 
 ### Behavior
 
--   Add a default preselected value using the recommended answer or, if there is none, the one most relevant to users.
--   Never use radio buttons if they are no answer possible in the set. The user **must** provide an answer to move on.
+-   Preselect the recommended option. If there is none, preselect the option most likely to be selected by users.
+-   Never use radio buttons if the set of options is not exhaustive. The user **must** select an option.
 
 ---
 
 ## Related Components
 
-### [Dropdown](#/components/DropdownSearch)
+Use a [Dropdown](#/components/DropdownSearch) when you have more than 7 options to display.
 
-Use Dropdowns when you have more than 7 options to show.
-
-### [Toggle](#/components/FlatSelect)
-
-Use Toggles for binary options with short labels (e.g. true or false), or for multiple options using abbreviated labels or acronyms (e.g. days of the week)
+Use a [Toggle](#/components/FlatSelect) for binary options with short labels (e.g., true or false), or for multiple options with abbreviated labels or acronyms (e.g., days of the week)
```

Comment:
For the Related Components section, it should be similar to Chuck's one, so we get rid of the description. 
Our plan is to eventually call the component description directly, so we don't have to maintain it everywhere it's called. 
So, if you could please change this only so we get only the link to the related component in bullet points, without the H3 style, that would be perfect.